config.reek

```diff
@@ -35,3 +35,8 @@ LongParameterList:
 # This one just makes sure the Class/Module has a comment. Dumb.
 IrresponsibleModule:
   enabled: false
+
+# Allow modules named "V2", "V1", etc
+UncommunicativeModuleName:
+  accept:
+    - !ruby/regexp /V[0-9]$/
```

Comment:
it compares the entire module name, example: "Napa::V1" so it could possibly be modified to /^|(::)V[0-9]$/